pesuacademy/models/staff.py

```diff
@@ -0,0 +1,27 @@
+class Staff:
+    def __init__(
+        self,
+        name: str,
+        designation: str,  
+        education: list,
+        experience: list,
+        campus: str,
+        department: str,
+        domains: list,
+        Responsibilities: list, 
+        mail : str
```

Comment:
General comment: If any of these is optional, remember to account for it

Reply:
added Optional for the required fields